src/assets/icons/Share.svg

```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
can you please add an export for this file in `src/assets/icons/index.tsx`?

Reply:
there is no `src/assets/icons/index.tsx` file but there is `src/assets/icons/index.js` so should I add the import to a `.js` file or create `src/assets/icons/index.tsx`?

Reply:
You're correct!
It the `.js` file.
I should have created a `.ts` file 🤔 
I'll fix that in a different PR.
Thanks!